ui/main/src/app/views/customCardList/ResultTable.ts

```diff
@@ -204,6 +212,9 @@ export class ResultTable {
                             data[column.field] = this.getNestedField(card, column.cardField);
                         }
                         break;
+                    case FieldType.HTML:
+                        data[column.field] = column.getValue(card);
+                        break;
```

Comment:
Add a unit test similar to "with getValue method if getValue is defined and fieldType is STRING'"